### PR TITLE
Searchbar

### PR DESCRIPTION
**Pull request by danicadombek**
Add searchbar to MemoriesPage with styling

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ export default function App() {
           onDetail={showDetailMemoryPage}
           onNavigateBack={showCameraPage}
           onDelete={handleDeleteMemory}
+          // onSubmit={choosenMemory}
```
Comment (frederikebruendermann) [inline src/App.js]: do you need this?

File: src/pages/MemoriesPage.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
 import DeleteIcon from '../assets/images/icons/delete.png'
 import cam from '../assets/images/icons/cam.png'
+import search from '../assets/images/icons/search.png'
+import { useState } from 'react'
```
Comment (frederikebruendermann) [inline src/pages/MemoriesPage.js]: Maybe you could organize your imports with cmd+shift+p :)

File: src/pages/MemoriesPage.js [hunk 3]
```diff
@@ -37,30 +41,68 @@ export default function MemoriesPage({
...
+                <MemoryItem
+                  image={image}
+                  title={title}
+                  date={date}
+                  text={text}
+                  id={id}
+                  onDetail={() => onDetail(image, title, date, text, id)}
+                />
+              </ListItem>
+            ))}
```
Comment (frederikebruendermann) [inline src/pages/MemoriesPage.js:92]: nice! maybe you could add a little hint if there is no result for the search :)

**Review by frederikebruendermann — APPROVED** [review]
Good work 🌼 I like how readable your code is! Just added some smaller suggestions and comments.

File: src/pages/MemoriesPage.js (hunk 2 above)
Comment (danicadombek) [reply src/pages/MemoriesPage.js]: I am going to organize it when I clean up the code XD

File: src/pages/MemoriesPage.js (hunk 3 above)
Comment (danicadombek) [reply src/pages/MemoriesPage.js:92]: Oh, good point!

**Pull request merged**